pkg/manifests/config.go

```diff
@@ -188,6 +189,17 @@ func (cfg *TelemeterClientConfig) IsEnabled() bool {
 	return true
 }
 
+// collectionProfilesAsString the list of collection profiles cannot easily be
+// formated using Sprintf since they are not of type string, this function
+// returns a string with all CollectionProfiles separated by comma
+func collectionProfilesAsString() string {
```

Comment:
suggestion: a more idiomatic way is to create a custom type and implement the Stringer interface for it:

```golang
type CollectionProfiles []CollectionProfile

func (cps CollectionProfiles) String() string {
        collectionProfiles := make([]string, 0, len(cps))
        for _, collectionProfile := range cps {
                collectionProfiles = append(collectionProfiles, string(collectionProfile))
        }
        return strings.Join(collectionProfiles, ", ")
}

...
// in NewConfig()
                return nil, errors.Wrap(ErrConfigValidation, fmt.Sprintf(`%q is not supported, supported collection profiles are: %q`, c.ClusterMonitoringConfiguration.PrometheusK8sConfig.CollectionProfile, SupportedCollectionProfiles.String()))

...

var SupportedCollectionProfiles = CollectionProfiles{FullCollectionProfile, MinimalCollectionProfile}

```

Reply:
Yes, this looks better! I implemented this